lib/utils/string_extension.dart

```diff
@@ -290,4 +290,14 @@ extension StringCasingExtension on String {
     );
     return result;
   }
+
+  String shortenDisplayName() {
+    if (length < 68) return this;
```

Comment:
use contants

Reply:
maybe add parameter `maxCharacter` so that you don't have to rewrite 2 methods

Reply:
done